style.css

```diff
@@ -737,16 +805,551 @@ ul {
+  height: 40px;
+  width: 305px;
   padding-left: 20px;
   border-radius: 8px;
   background: #fff;
   border: 1px solid #cfd8dc;
-  font-family: "Poppins", sans-serif;
   font-size: 17px;
   font-weight: 400;
   line-height: 20px;
   margin-bottom: 20px;
   color: #172b4d;
 }
+
+#text {
+  display: flex;
+  height: 160px;
+  width: 305px;
+  border-radius: 8px;
+  margin-bottom: 20px;
+  padding-left: 20px;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #172d4d;
+}
+
+#contact-submit {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 48px;
+  width: 150px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  padding: 12px 16px 12px 16px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  margin-top: -20px;
+}
+
+#contact-submit:hover {
+  background-color: #6070ff;
+  font-size: 18px;
+  color: #fff;
+}
+
+#contact-submit:active {
+  background-color: #2230d2;
+  font-size: 18px;
+  color: #fff;
+}
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    overflow-x: hidden;
+    background-color: #fff;
+  }
+
+  .logo {
+    font-size: 25px;
+    font-weight: 700;
+    line-height: 20px;
+    background-color: transparent;
+    transition: width 2s, height 2s, background-color 2s, rotate 2s;
+    rotate: 360deg;
+  }
+
+  .app-header {
+    display: flex;
+  }
+
+  .humbourger {
+    display: none;
+  }
+
+  .ambony {
+    width: auto;
+  }
+
+  .menu {
+    display: flex !important;
+    justify-content: flex-end;
+    align-items: center;
+    margin-right: 20px;
+  }
+
+  .menu-item {
+    margin-right: 20px;
+  }
+
+  .menu-item a {
+    display: flex;
+    font-size: 1rem;
+    color: #344563;
+    text-decoration: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    text-align: center;
+  }
+
+  .header-ambony {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    width: 100%;
+    padding-left: 10%;
+    background-color: #fff;
+    background-image: url(media/Header\ bg.svg);
+    background-size: cover;
+    top: 2.875rem;
+    border-bottom-left-radius: 60px;
+    height: 44.88rem;
+  }
+
+  .header-container {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding-left: 10%;
+  }
+
+  .narson {
+    width: 100%;
+  }
+
+  .detail {
+    height: 98px;
+    width: 920px;
+    left: 0;
+    top: 134px;
+  }
+
+  .connection {
+    margin-top: -1px;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+    text-align: left;
+    color: #7f8cff;
+  }
+
+  /* work section */
+
+  .work {
+    display: grid;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 86.5%;
+    margin: 10px 0 0 28px;
+    height: auto;
+    background-color: #fff;
+  }
```

Comment:
Great job implementing the change of the previous reviewer and remove the two background color. 
- [ ] Please update the background color of the work section to match the figma design.
**Your project**
![image](https://user-images.githubusercontent.com/74411135/230723826-14b27182-2491-44d2-92c6-134a42ce78ac.png)
**figma design**
![image](https://user-images.githubusercontent.com/74411135/230723841-3b090bd3-6756-48ec-8936-6954616d5178.png)

This will make your project prettier and make the user know you're an awesome developer as you can precisely follow a professional web developer's design 😃 💪